### ISSUE TITLE
Can't Generate a Legacy Wallet?

### ISSUE THREAD

**Issue by S0yf00d-exe** [pooler/electrum-ltc#299]
### [ Wallet Version: 4.2.2.1 AppImage ]

So when folloing the steps to create a new wallet there is no option after selecting *Standard Wallet* that prompts you if you want it to be segwit or legacy like the Electrum BTC Wallet does.

I looked around and found in https://github.com/pooler/electrum-ltc/issues/170 and https://github.com/pooler/electrum-ltc/issues/236 some things...

In issue #170 it mentions a converter:

>https://litecoin-project.github.io/p2sh-convert/  

I tried my standard wallet legacy adress starting with ltc 

**(no uppercase and lowers, just lowercase whole address)** 

After putting Address in converter I get *"please enter a valid address."*

In issue #236

@pooler Pooler (Febuary 2020) says:

> When creating a wallet you have to choose between Segwit and Legacy, and Electrum-LTC shows this warning:

> Segwit wallets use bech32 addresses, defined in BIP173. Please note that websites and other wallets may not support these addresses yet. Thus, you might want to keep using a non-segwit wallet in order to be able to receive litecoins during the transition period.

**This never comes up for me and I am on latest appimage** ^ (as mentioned above) ^

I verified the GPG signature verification of the wallet following steps on site so the signature was *good*.

## Anyway can you no longer generate Legacy wallets in Electrum-ltc wallet?

If you can what is the workaround or what am I doing wrong?

I see a console (terminal) option in wallet so maybe that is it?

**Comment by pooler** [comment]
Since version 4.1.0 of Electrum, the wallet creation wizard no longer asks for a seed type, and creates segwit wallets with bech32 addresses. Older seed types can still be created with the command line, or via the console tab with:

```
make_seed(seed_type="standard")
```

This will display a new seed which you can then use to "restore" a fresh legacy wallet.

**Comment by S0yf00d-exe** [comment]
@pooler

So I used the console and put in command `make_seed(seed_type="standard")`and it generated new seed.

Then proceded to create new wallet and imported/restored with seed generated from console and it worked. 👍 

 [Created Address with Uppers and Lowers case] aka Legacy

**Thanks**

*Is this console command documented on the site?* 

In Closing the issue, if its not documented `make_seed(seed_type="standard")` it should be added on site and Wiki to safe people the hasle when trying to buy *LTC* and address not working cause its not Legacy to begin with.

**Comment by pooler** [comment]
I agree this should probably be better documented, even upstream. Googling the issue mostly returns results for unofficial (although helpful) pages:
https://bitcoinelectrum.com/creating-an-electrum-wallet/
https://www.reddit.com/r/Electrum/comments/mku14h/how_do_you_create_legacy_wallet_from_command_line/

I think it might even be a good idea to revert the change to the wallet creation wizard in future releases of Electrum-LTC, given how bech32 addresses are still not widely supported.

**Comment by cmonty14** [comment]
> make_seed(seed_type="standard")

Hi,
can you please advise how to create a wallet of older seed types with CLI or console tab?

THX
